src/tx/builder/schema.ts

```diff
@@ -321,10 +363,19 @@ export const TX_FEE_OTHER_GAS = (txType, txSize, { relativeTtl, innerTxSize }) =
   }
 }
 
+export interface TxBase {
+  tag?: number
+  vsn?: number
+  gasLimit?: number
+  fee?: BigNumber | number | string
+  ttl?: BigNumber | number | string
+  nonce?: number | BigNumber | string
```

Comment:
Not sure why the base interface have more fields than the base schema (defined below)